src/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -362,7 +364,11 @@ void PreStepsList::add_reverse_channels() {
     m_actions.emplace_back([](const std::vector<Output<Node>>& nodes,
                               const std::shared_ptr<Model>& function,
                               PreprocessingContext& context) {
-        return reverse_channels(nodes, function, context);
+        auto resp = reverse_channels(nodes, function, context);
+        auto outputs = std::get<0>(resp);
+        OPENVINO_ASSERT(outputs.size() == 1, "Internal error: reverse_channels returned unexpected number of outputs");
+        set_is_reverse_input_channels(outputs.at(0).get_node_shared_ptr());
```

Comment:
is it ok? we pass nodes here. maybe there are several RICs there?

Reply:
Yes, it is ok.  https://github.com/openvinotoolkit/openvino/blob/231dcb75f5c084900a375c6c71c3cb056db0e39e/src/core/src/preprocess/preprocess_steps_impl.cpp#L378